rollup/internal/utils/exchange_rate.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+
+	"github.com/scroll-tech/go-ethereum/log"
+)
+
+var BinanceApiEndpoint string = "https://api.binance.com/api/v3/ticker/price?symbol=%s"
+
+type BinanceResponse struct {
+	Price string `json:"price"`
+}
+
+func GetExchangeRateFromBinanceApi(tokenSymbolPair string) (float64, error) {
```

Comment:
maybe add some retry logic of get BinanceApiEndpoint

Reply:
thanks, added retry logic.